### ISSUE TITLE
[BUG] Card does not refresh over time

### ISSUE THREAD

**Issue by Johmpa** [marksie1988/atomic-calendar-revive#158]
### Prerequisites

- [X] Put an X between the brackets on this line if you have done all of the following:
  - Checked the Quick Start documentation page and confirmed configuration is correct: <https://marksie1988.github.io/atomic-calendar-revive/quickstart.html>
  - Checked the FAQs on documentation page for common solutions: <https://marksie1988.github.io/atomic-calendar-revive/faq.html>
  - Checked that your issue isn't already filed: <https://github.com/marksie1988/atomic-calendar-revive/issues?q=is%3Aissue>

### Describe the bug

The card state does not refresh over time. Events are not marked as completed, days do not go forward, progressbars do not progress, new events do not show up, no loading is performed etc. The only way to update is to refresh the entire page or navigate away from the dashboard. 

This worked fine on the original atomic-calendar implementation so something is amiss, although I cannot tell if it is my environment that is to blame. This has been observed on Chrome browser on multiple computers, all running Windows 10. 

### Steps to Reproduce

1. Create a card with a calendar
2. Show the lovelace dashboard with the card
3. Create an event in the calendar. The event will not show up by itself
4. Refresh the page and wait until the event starts. A progressbar will not appear
5. Wait until the event completes. It will not grey out. 

**Expected behavior**

New events should be loaded promptly, events should properly be shown as completed, days should reflect actual day by itself etc.

**Actual behavior:**

The lovelace card is effectively frozen in the state it loaded as until a refresh is forced. 

**Screenshots**

**Card Configuration**

```  
  - disableEventLink: true
    disableLocationLink: true
    entities:
      - entity: calendar.events
      - blacklist: 'Daily, Lunch, synk'
        entity: calendar.work
      - entity: calendar.personlig
      - entity: calendar.releasecalendar
      - entity: calendar.warhammer
      - entity: calendar.helgdagar_i_sverige
      - entity: calendar.vuxna_man_gor_saker_tillsammans
    language: sv
    maxDaysToShow: 1
    noEventsForNextDaysText: Ingenting kommande dagar
    noEventsForTodayText: Ingenting idag
    showColors: true
    showDate: true
    showFullDayProgress: true
    showLoader: true
    showLocation: false
    showNoEventsForToday: true
    showProgressBar: true
    showWeekDay: true
    sortByStartTime: true
    title: Kalender
    type: 'custom:atomic-calendar-revive'
```

### Versions

- HomeAssistant Version: 0.110.3
- Plugin Version: 1.8.3

**Comment by marksie1988** [comment]
The card will refresh every 600 seconds if there has been an update to the calendar.

**Comment by Johmpa** [comment]
That has not been my experience. I have added test-events in my calendars to narrow down the behavior and these events do not show up several hours afterwards. 

I have also verified that the card does not update the active day either. This Sunday for instance, it was still showing Saturday as the current day, with several events showing as halfway complete even though they were long since finished. The only way to get the card state to update is doing a refresh of the dashboard page. 

This all works fine still in the previous atomic calendar incarnation as I've run it in parallel for comparison.

**Comment by marksie1988** [comment]
Ok thanks for the input, I've just released a new version that makes some changes to the way updates are triggered, may be worth trying that to see if it resolves your issue.

**Comment by Johmpa** [comment]
This did indeed solve the issue, thanks for the quick response! I have noticed that this release seem to have affected the progressbars though - they are always completely full. both on all-day events and standard events. 

That is out of scope of this issue so I regard this one as solved.
